examples/plot_simulate_mpi_backend.py

```diff
@@ -0,0 +1,72 @@
+"""
+=======================================
+06. Use MPI backend for parallelization
+=======================================
+
+This example demonstrates how to use the MPI backend for
+simulating dipoles using HNN-core.
+
+The MPI backend allows running the simulation in parallel across neurons in the
+network even with a single trial. For this, you will
+need the :ref:`MPI related software <parallel>` installed. Note that if you
+want to simulate in parallel across trials, the Joblib backend allows this
+without the need to install and configure MPI.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Blake Caldwell <blake_caldwell@brown.edu>
+
+###############################################################################
+# Let us import hnn_core
+import os.path as op
+
+import hnn_core
+from hnn_core import simulate_dipole, read_params, Network
+
+###############################################################################
+# Then we setup the directories and Neuron
+hnn_core_root = op.dirname(hnn_core.__file__)
+
+###############################################################################
+# Then we read the default parameters file
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+print(params)
+
+###############################################################################
+# Update a few of the default parameters related to visualisation
+params.update({
+    'dipole_scalefctr': 150000.0,
+    'dipole_smooth_win': 0,
+    'tstop': 310.0,
```

Comment:
In line with @cjayb's comments on #262. the scale factor and smooth window can be taken care of with `simulate(postproc=False)` and `dpl[trial_idx].scale()`. Unless you want to remove the scaling here as well